### ISSUE TITLE
About FairMOT results

### ISSUE THREAD

**Issue by KevinKai-0717** [ifzhang/FairMOT#207]
I noticed that FairMOT updated the evaluation results on the MOTChallenge platform and refreshed the optimal results on multiple datasets. Is this an update based on FairMOT or a new algorithm? Does FairMOT code have an update plan? Best wishes for you!

**Comment by ifzhang** [comment]
Yes, I will update the code and model when I finish cleaning the code. There's not much changing of the code. I just change some parts of the data processing code.

**Comment by KevinKai-0717** [comment]
Looking forward to your update！Best wishes for you!

**Comment by khodabakhshih** [comment]
count down for the new update. Thanks
